contexts/did-v0.11.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+    "@context": {
+      "@version": 1.1,
```

Comment:
nit: indentation is wrong

Reply:
won't fix... we can add a lint process and precommit hook to address this in the future... https://github.com/w3c/did-core-registry/issues/16